### ISSUE TITLE
Change some Latin characters. Opinion?

### ISSUE THREAD

**Issue by kiwi0fruit** [alerque/libertinus#133]
I'd like to change some Latin characters.
    * Adapt <>, ≤≥ from [Crimson](https://fontlibrary.org/en/font/crimson) (Check other symbols like <>),
    * Change italic g to the one like in Computer modern (single-story): make g from a and ɡ. Bold and semibold make from a and 5 (also check ʒ). I think single-story g is better for mathematical contexts. It would look like the 1st one:
    
![](https://user-images.githubusercontent.com/19735117/36479036-c6d0e8cc-1739-11e8-9e19-9ee610484c78.jpg)

What do you think about it? Would you merge it?

**Comment by alerque** [comment]
As a publishing house with print works released in both _Libertinus_ and _Crimson_, I would  not be too keen on a switch to a singe story `g`. In fact I would actively vote against that. If a a single story version is better for math contexts that could be done in _Libertinus Math_, but please don't touch _Libertinus Serif_. I also think the respective numbers are a better match for their respective fonts as they are now. I don't see any reason to swap them out.  I just had a look at `@` and like the _Crimson_ one but don't see a need to change the _Libertinus_ one either.

**Comment by kiwi0fruit** [comment]
Actually numbers are the last in line :) Just thoughts...

@ is rather easy to change 

The single story g is the main interest.

**Comment by khaledhosny** [comment]
We already have a single storey g glyph for ɡ that is used in IPA, if math users really prefer a single storey math g, we can switch to that glyph. Though I checked both STIX and STIX Two math fonts and they use double storey italic g (STIX Two uses a single storey one for the bold italic, though), so that does not seem like a universal preference, so I’d add the single storey g as an alternate.

For the other suggestions I agree with what @alerque said.

**Comment by kiwi0fruit** [comment]
It's not g actually in the IPA - see the right glyph in the top post picture - that's it. The proper italic g should look like the left glyph on that pic.

**Comment by kiwi0fruit** [comment]
And I it's better to look at Cambria Math and Computer Modern - I think they are more popular than STIX. They both have single story italic g.

**Comment by khaledhosny** [comment]
Both Cambria Math and Computer Modern are in style too far away from Libertinus, however STIX is far closer to it so it is an appropriate comparison IMO. I don’t have a string objection about you proposed g, but I don’t want it to be the default.

**Comment by kiwi0fruit** [comment]
I think they are close to Libertinus than to each other. But both have single story g. It's not about font style. It's about when you write g(x) on paper it's always single story. And math font inherits this.

**Comment by kiwi0fruit** [comment]
But still Libertinus Serif is a transitional font. And most of the transitional fonts have single story g (except Georgia and some other)...

**Comment by kiwi0fruit** [comment]
Oops. Most have double story g.

**Comment by kiwi0fruit** [comment]
Single/double story aside: I think that italic g from Libertinus Serif is too similar to regular. For comparison here are some screenshots from nice [math font review](http://math.sut.ac.th/lab/software/texlive/texmf-doc/doc/english/free-math-font-survey/survey.pdf):
(first one is Libertinus, others - clearer italic g that can't be confused with regular one)

![screenshot_20180222-005056 2](https://user-images.githubusercontent.com/19735117/36496559-3a662172-176b-11e8-98ba-befb19a43793.png)
![screenshot_20180222-004922 2](https://user-images.githubusercontent.com/19735117/36496564-3f6e2318-176b-11e8-8026-613e7f14fe71.png)
![screenshot_20180222-004836 2](https://user-images.githubusercontent.com/19735117/36496569-45407f48-176b-11e8-9d0b-7b8bdcf47e65.png)

**Comment by kiwi0fruit** [comment]
@khaledhosny Would you pull it if I implement it as a stilistic set: https://helpx.adobe.com/typekit/using/open-type-syntax.html#ss## ?

**Comment by khaledhosny** [comment]
Yes.

**Comment by jabirali** [comment]
I agree, having a single-story g as a stylistic alternate in Libertinus Math would be great! For the record, this is exactly what STIX 2.0 does as well, here's an excerpt from their documentation:
![screenshot at 2018-04-22 19-21-23](https://user-images.githubusercontent.com/271284/39097871-6c0d6822-4662-11e8-89fa-a2f0cb8f71b1.png)

**Comment by Bolpat** [comment]
Can't wait to see it. The :+1: was meant for Libertinus Math. Question aside: Is there a v in Libertinus Math like STIX default?

**Comment by khaledhosny** [comment]
I think the question is already answered; a pull request to add an optional alternate italic g for math is welcomed.
